seed/python-sdk/api-wide-base-path/README.md

```diff
@@ -57,6 +57,69 @@ asyncio.run(
 )
 ```
 
+## Exception Handling
+
+When the API returns a non-success status code (4xx or 5xx response), a subclass of the following error
+will be thrown.
+
+```python
+from .api_error import ApiError
+
+try:
+    client.service.post()
+except ApiError as e:
+    print(e.status_code)
+    print(e.body)
+```
+
+## Retries
+
+The SDK is instrumented with automatic retries with exponential backoff. A request will be retried as long
+as the request is deemed retriable and the number of retry attempts has not grown larger than the configured
+retry limit (default: 2).
+
+A request is deemed retriable when any of the following HTTP status codes is returned:
+
+- [408](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408) (Timeout)
+- [429](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) (Too Many Requests)
+- [5XX](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) (Internal Server Errors)
+
+Use the `max_retries` request option to configure this behavior.
+
+```python
+client.service.post({
+    max_retries=1
+})
+```
+
+## Timeouts
+
+The SDK defaults to a 60 second timeout. You can configure this with a timeout option at the client or request level.
+
+```python
+
+from seed.client import SeedApiWideBasePath
+
+client = SeedApiWideBasePath(..., { timeout=20.0 }, )
+
+
+# Override timeout for a specific method
+client.service.post({
+    timeout_in_seconds=1
+})
+```
+
+## Custom Client
+
+You can override the `httpx` client to customize it for your use-case. Some common use-cases include support for proxies
+and transports.
+```python
+import httpx
+from seed.client import SeedApiWideBasePath
+
+client = SeedApiWideBasePath(..., http_client=httpx.Client(proxies=http://my.test.proxy.example.com, transport=httpx.HTTPTransport(local_address="0.0.0.0"), ), )
```

Comment:
weird that this is how it gets formatted -- is this affected by the trailing comma?

Reply:
Right? I'm actually not sure why it gets formatted this way, I need to see what's happening here

Reply:
Fixed!